pkg/jobrunaggregator/jobruntestcaseanalyzer/analyzer.go

```diff
@@ -41,6 +41,7 @@ type testCaseAnalyzerJobGetter struct {
 	platform       string
 	infrastructure string
 	network        string
+	filters        sets.String
```

Comment:
All the other criteria are positive criteria, this is the first excluding criteria.  That seems worth a comment or a name like `exclusionaryFilters`

Reply:
does `excludeJobNames` work?